clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheStateServiceImpl.java

```diff
@@ -242,6 +250,139 @@ private ResourcePageSource createPageSource(String poolName, ServerSideConfigura
     return pageSource;
   }
 
+  private void registerStoreStatistics(String storeName) throws InvalidStoreException {
```

Comment:
All of this should leverage Java 8 constructs and if still large be moved to a companion object.

For example, all the invocations of `registerStatistic` change by two params - the stat name and the method invoked in the callable. By having a map of these and writing some functional code, all this could be done in a more compact way.